### ISSUE TITLE
Run aXe on generated sites

### ISSUE THREAD

**Issue by toolness** [18F/federalist#1113]
This is similar to #1112, but would run the [axe-core](https://github.com/dequelabs/axe-core) accessibility tool on at least the front page of the generated site. This can help alert Federalist users to potential 508 violations on their sites.

Implementation-wise, this is a bit of a heavier lift than #1112 because it requires the generated site to be visited in a headless browser (in contrast, the link checker is run on a page's raw HTML content, not in a browser context, so it's very fast).

More advanced integration might allow Federalist users to specify a list of pages to visit with aXe, or even tell federalist to just crawl the entire site and run aXe on every page (this could take a while).

**Comment by toolness** [comment]
Oh, @maya just pointed me at #1062, which is very related!

**Comment by wslack** [comment]
Closing in favor of https://github.com/18F/federalist/issues/1062 (copied the content)
